### PR TITLE
fix(pnr): add top attribute on ui-kit elements

### PR DESCRIPTION
**Pull request by Jisay**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/product-nav-reshuffle`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-9363
| License          | BSD 3-Clause

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/account/user/dashboard/user-dashboard.html [hunk 1]
```diff
@@ -97,6 +97,7 @@
                         <oui-action-menu data-compact data-placement="end">
                             <oui-action-menu-item
                                 data-href="#/useraccount/infos"
+                                top="true"
```
Comment (cbourgois) [inline packages/manager/apps/dedicated/client/app/account/user/dashboard/user-dashboard.html:100]: Seems to redirect from `dedicated` to `dedicated`. 
Not external and not `buildURL`.

File: packages/manager/apps/web/client/app/domain/general-informations/GENERAL_INFORMATIONS.html [hunk 2]
```diff
@@ -68,6 +68,7 @@
                     <oui-action-menu data-compact data-placement="end">
                         <oui-action-menu-item
                             data-href="{{ ::$ctrl.actions.manageAlldom.href }}"
+                            top="true"
```
Comment (cbourgois) [inline packages/manager/apps/web/client/app/domain/general-informations/GENERAL_INFORMATIONS.html:71]: Are theses URLs external or results of `$state.href` ?

File: packages/manager/apps/web/client/app/hosting/freedom/FREEDOM.html [hunk 3]
```diff
@@ -116,6 +116,7 @@ <h2 data-translate="hosting_tab_FREEDOM"></h2>
                             <oui-action-menu-item
                                 data-on-click="ctrlHostingFreedomTab.setSelectedProduct(freedom.domain)"
                                 data-href="{{'#/domain/'+freedom.domain}}"
+                                top="true"
```
Comment (cbourgois) [inline packages/manager/apps/web/client/app/hosting/freedom/FREEDOM.html:119]: Not required. We need to use `_top` only for external URL or url builded with URLBuilder
We need to check more in details.

**Review by cbourgois — CHANGES_REQUESTED** [review]

**Comment by Jisay** [comment]
I have checked all the links and added `top` attribute to all of them that leaved the application.

**Review by cbourgois — APPROVED** [review]

**Pull request merged**